app/globals.css

```
@@ -1,5 +1,15 @@
 @import "tailwindcss";
 
+html {
+  scroll-behavior: smooth;
+}
+
+`@media` (prefers-reduced-motion: reduce) {
```

Stylelint (17.12.0)

[error] Unknown type selector "`@media`" (selector-type-no-unknown)

(selector-type-no-unknown)